client/config.go

```diff
@@ -153,27 +153,24 @@ func (c Configuration) NewAdminClient(
 
 	var err error
 	if envCfg.TopologyInitializer == nil {
-		switch {
-		case c.EnvironmentConfig.Service != nil:
-
+		if c.EnvironmentConfig.Service != nil {
```

Comment:
Just out of curiosity, what's the reasoning for switch to if change?

Reply:
Purely stylistic. There was a previous PR comment noting that using switch here felt a little odd, which I agreed with.